src/io/flutter/test/DartTestEventsConverterZ.java

```diff
@@ -730,6 +730,13 @@ public String toString() {
   }
 
   protected static class Test extends Item {
+
+    private static class LocationInfo {
```

Comment:
nit: it would be slightly cleaner to make all fields of this class final with a constructor that specifies all fields.

Reply:
In general I totally agree.  In this case since it's private and essentially a struct I didn't though.  Plus it would necessitate a bunch of locals to hold on to intermediate values.  But yeah, I'll let it simmer. 👍